restful_api/myapi/api/schemas/self_enroll.py

```diff
@@ -0,0 +1,13 @@
+from myapi.models import SelfEnroll
+from myapi.extensions import ma, db
+
+
+class SelfEnrollSchema(ma.SQLAlchemyAutoSchema):
+    
+    eng_id = ma.Int(dump_only=True)
+    course_id = ma.Int(dump_only=True)
```

Comment:
Does it work without these lines? Technically, the information is inside `model = SelfEnroll` already